dbclient/MLFlowClient.py

```diff
@@ -179,8 +155,21 @@ def _create_experiment(self, experiment_str, id_map_writer, checkpointer, error_
             new_id = self.client.create_experiment(name, artifact_location, dict_tags)
             logging.info(f"Successfully created experiment with name: {name}. id: {new_id}")
         except RestException as error:
-            logging.info(f"create experiment failed for id: {id}, name: {name}. Logging it to error file..")
-            error_logger.error(error.json)
+            # If the resource already exists, then we can consider it successful and checkpoint it.
+            if error.json['error_code'] == 'RESOURCE_ALREADY_EXISTS':
+                logging.info(error.json['message'] + f" Trying to get the experiment_id of the existing experiment: {name}...")
+                try:
+                    new_id = self.client.get_experiment_by_name(name).experiment_id
+                    logging.info(f"Successfully retrieved an id: {new_id} for experiment: {name} in the target workspace.")
```

Comment:
We should also just make sure to set tags in this case, e.g. for notebook experiments

Reply:
done!